src/workingWithData.ts

```diff
@@ -0,0 +1,54 @@
+import { v4 as uuidv4 } from 'uuid'
+import { resolve } from 'path'
+import { readFile, open, writeFile, unlink, access, mkdir } from 'fs/promises'
+
+type UserInfoType = { name: string; age: number }
```

Comment:
You can move this type to another directory. For example you can create directory types and file user.type.ts and put there your type. Also you can put there validators

Reply:
Done